files/en-us/learn/server-side/express_nodejs/deployment/index.html

```diff
@@ -189,8 +196,9 @@ <h3 id="Use_gzipdeflate_compression_for_responses">Use gzip/deflate compression
 ...
 </pre>
 
-<div class="note">
-<p><strong>Note</strong>: For a high-traffic website in production you wouldn't use this middleware. Instead, you would use a reverse proxy like <em>Nginx</em>.</p>
+<div class="notecard note">
+  <h4>Note</h4>
+  <p>For a high-traffic website in production you wouldn't use this middleware. Instead, you would use a reverse proxy like <em>Nginx</em>.</p>
```

Comment:
```suggestion
  <p>For a high-traffic website in production you wouldn't use this middleware. Instead, you would use a reverse proxy like <em>nginx</em>.</p>
```
Looking through https://nginx.org/en/docs/faq.html it appears, that `nginx` is the product and `Nginx` is the company (but the literature is inconsistent here).